CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+* `FSUrlLoader#canLoad` reports false for local urls outside the loader's
+  own root; enables fall-thru support needed for racking in `MultiUrlLoader`.
```

Comment:
racking?

Reply:
Yeah that wasn't clear.  I'll rename this to sequencing.

Reply:
Better yet I just rewrote "support need for use with `MultiUrlLoader`"